approval_polls/templates/approval_polls/poll_update_form.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class='row-fluid top-buffer'>
+        <div class='col-md-12'>
+            <h3> Edit Poll </h3>
+        </div>
+    </div>
+    <div class='row-fluid top-buffer'>
+        <div class='col-md-12'>
+            <form action="" method="post"> {% csrf_token %}
```

Comment:
Double quotes will do. I use pycharm and it single quotes by default. 

Blank action. Good question. By default a blank action simply means "post back to your self" as it were. In general I've learned it is a good practice to have the action there and blank rather than missing all together.